test/performance/metrics_collector.py

```diff
@@ -98,29 +126,68 @@ def check_pod_status(self, namespace, watch_interval, label=None):
                         except ApiException as e:
                             print("Exception when calling CoreV1Api->delete_namespaced_pod: %s\n" % e)
 
-                        if c == self.num_objects:
+                        if c == numPods:
                            print("Total number of pods COMPLETED: ", c)
                            w.stop()
                            break
+            f1.close()
+            f2.close()
+            f3.close()
+            f4.close()
+            f5.close()
         except ApiException as e:
             print("Exception when calling CoreV1Api->list_namespaced_pod: %s\n" % e)
 
+
+    def collect_short_exp_metrics(self, path, num_ns, ns_prefix=None, wds_ctx=None, wec_ctx=None):
+
+        obj_kind_list = ["deployments", "secrets", "configmaps", "services"]
+
+        for i in range(0, num_ns):
+            ns = ns_prefix + "-" + str(i)
+            dir_ns = path + ns
+
+            if not os.path.exists(dir_ns):
+               os.makedirs(dir_ns)
+
+            for kind in obj_kind_list:
+                fname = dir_ns + "/" + kind
+                self.collect_obj_time(self.c1, None, kind, fname + "-wds1", ns)
+                self.collect_obj_time(self.c3, None, kind, fname + "-wec", ns)
+
+            # collect manifestWork, workStatus and appliedManifestWork objects    
+            bindingPolicy = ns
+            self.collect_obj_time(self.c2, None, 'cr', dir_ns + "/workstatuses", wec_ctx, "control.kubestellar.io", "v1alpha1", "workstatuses", bindingPolicy)
+            self.collect_obj_time(self.c2, None, 'cr', dir_ns + "/manifestworks", wec_ctx, "work.open-cluster-management.io", "v1", "manifestworks", bindingPolicy)
+            self.collect_obj_time(self.c3, bindingPolicy + '-' + wds_ctx, 'cr', dir_ns + "/appliedmanifestworks", None, "work.open-cluster-management.io", "v1", "appliedmanifestworks", None, True, False)
+
 if __name__=="__main__": 
   
-    kubeconfig = "kscore-config"
-    ns = "stress-test"
-    mypath="dat/"
-    run_ID="1"
-    freq="1"
-    numPods=20
-    watch_interval = 3600
-    method = "exp-RandomizedLoad"
-
-    f1 = mypath + method + "-" + "freq-" + str(numPods) + "-pod-wds-kscore-stage-run" + run_ID + ".txt"
-    f2 = mypath + method + "-" + "freq-" + str(numPods) + "-manifestwork-its1-run" + run_ID + ".txt"
-    f3 = mypath + method + "-" + "freq-" + str(numPods) + "-workstatus-its1-run" + run_ID + ".txt"
-    f4 = mypath + method + "-" + "freq-" + str(numPods) + "-pod-wec1-run" + run_ID + ".txt"
-    f5 = mypath + method + "-" + "freq-" + str(numPods) + "-appliedmanifestwork-wec1-run" + run_ID + ".txt"
-
-    c = DataParser(numPods, kubeconfig, 'wds1', 'its1', 'wec1', f1, f2, f3, f4, f5)
-    c.check_pod_status(ns, watch_interval)
+    first = int(sys.argv[1])
+    second = int(sys.argv[2])
```

Comment:
These two are not used for anything, right? Why are they here?

Reply:
Oh! forgot to clean up those parameters - Done!